package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@playkit-js/playkit-js-downloads",
   "version": "1.3.8",
-  "main": "dist/playkit-downloads.js",
+  "main": "playkit-js-downloads",
```

Comment:
why ? the file which webpack outputs is named playkit-downloads, no ?

also the filename from webpack config will affect the plugin name, and this plugin is already in production, so changing it will break backward compatibility